db-mongo/seed.js

```diff
@@ -1,53 +1,54 @@
 
-const insertSampleData = function (type = 'hipster-centric', paras = 5) {
-  const randomNum = (max) => {
-    return Math.floor(Math.random() * Math.floor(max))
-  }
+let count = 0;
+const insertSampleData = (type = 'hipster-centric', paras = 5) => {
+  const randomNum = (min, max) => Math.floor(Math.random() * (max - min) + min);
   const sampleReviews = [];
 
-  const url = `https://hipsum.co/api/?type=${type}&paras=${paras}`
+  const url = `https://hipsum.co/api/?type=${type}&paras=${paras}`;
   fetch(url)
-  .then(response => response.json())
-  .then(data => {
-    data.forEach((x) => {
-        let productId = count < 10 ? `P00${count}` : count > 9 && count < 100 ? `P0${count}` : `P${count}`
+    .then((response) => response.json())
+    .then((data) => {
+      data.forEach((x) => {
+        const productId = count < 10 ? `P00${count}` : count > 9 && count < 100 ? `P0${count}` : `P${count}`;
```

Comment:
@cpalka87 for double-ternaries i recommend breaking them into separate variables that explain what your logic is in words